config/locales/en/views/bulk_submissions.yml

```diff
@@ -13,3 +17,13 @@ en:
       process_all: Process all pending
       remove_html: Remove <span class='govuk-visually-hidden'>%{filename}</span>
       status: Status
+    show:
+      confirm_cancel_html: Confirm cancel <span class='govuk-visually-hidden'>%{filename}</span>
+      confirm_remove_html: Confirm remove <span class='govuk-visually-hidden'>%{filename}</span>
```

Comment:
nit: could we make this content clearer?

for example:
"Confirm cancellation of filename" instead of "Confirm cancel filename"
"Confirm removal of filename" instead of "Confirm remove filename"

Reply:
content design applied now